logging/include/LoggingSpd.hpp

```diff
@@ -0,0 +1,93 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the
+// LICENSE file.
+
+#pragma once
+
+#include <string>
+#include <sstream>
+#include "spdlog/spdlog.h"
+
+#ifdef USE_SPDLOG
+
+namespace logging {
+
+// typedef log4cplus::Logger Logger;
+using Logger = std::shared_ptr<spdlog::logger>;
+
+std::string get(const std::string& key);
+
+}  // namespace logging
+
+#define LOG_TRACE(l, s)                        \
```

Comment:
There is a LOG4CPLUS_TRACE/DEBUG etc family of macros. I think those also need to be taken care of.

Reply:
So, if USE_LOG4CPP is not defined, then also the LOG4CPLUS_* macros are wrapped into LOG_*, since I see there definition only inside Logging4cplus.hpp

Reply:
This is used only in daml, so we are good.